public/API/API_GetConsoleIDs.php

```diff
@@ -9,6 +9,15 @@
  *   object    [value]
  *    string    ID                  unique identifier of the console
  *    string    Name                name of the console
+ *    string    IconURL             system icon URL
  */
 
-return response()->json(System::select(['ID', 'Name'])->get());
+$systems = System::all()->map(function ($system) {
+    return [
+        'ID' => $system->ID,
+        'Name' => $system->Name,
+        'IconURL' => $system->icon_url,
```

Comment:
I imagine it's desirable to return the full URL
```
"IconURL":"http:\/\/localhost:64000\/assets\/images\/system\/2600.png"
```
but other URLs in the public API are relative:
```
"BadgeName":"385859","BadgeURL":"\/Badge\/385859.png"
```
```
"GameIcon":"\/Images\/085573.png","ImageIcon":"\/Images\/085573.png","ImageTitle":"\/Images\/054993.png","ImageIngame":"\/Images\/000010.png","ImageBoxArt":"\/Images\/051872.png"
```

Reply:
Yes, as @luchaos mentioned before, full URL is desirable 👍

Reply:
Oh, I missed that conversation - it's been marked as resolved.